plugins/GrapesJsBuilderBundle/Demo/mautic/index.html

```diff
@@ -2,6 +2,7 @@
   let mauticLang = 'en';
   let mauticBasePath = '';
   let mauticEnv = 'dev';
+  var mauticBaseUrl = 'http://mautic.ddev.site/index_dev.php/';
```

Comment:
The index_dev.php part will not work on production. Is that OK?

Reply:
yes, this is just for local development and debugging

Reply:
Ok, so this is not the production code. I'm a bit afraid that this will not work for developers who do not use ddev. Can we use something like
```suggestion
  var mauticBaseUrl = window.location.origin + '/index_dev.php/';
```

Reply:
unfortunately not. The JS code runs on its own server on localhost:1234. 

I documented it in the readme of the plugin. But it got replaced:
https://github.com/mautic/mautic/tree/4.x/plugins/GrapesJsBuilderBundle

Here is "a backup":
https://github.com/adiux/plugin-grapesjs-builder#how-to-test-standalone

I think this readme is actually quite important and we should make it available somewhere again. I would like to have it in the readme. Maybe in the preset repo. But I guess Mautic would like to have it in the dev docs?